### DISCUSSION TITLE
Debugger in Lapce?

### DISCUSSION THREAD

**Discussion by donatengit** [lapce/lapce#1438] [Ideas]
Hi Guys,

Appreciate your work.

I'm new to Rust, solving problems on Leetcode and I love the speed (especially on huge inlined test cases) and look&feel of the editor. The one thing I wasn't able to achieve with Lapce is some kind of debugger which comes very handy in complex cases. Like CoreLLDB plugin in VSCode.
Any advice how something similar could be enabled?

Thanks in advance

**Comment by panekj** [comment]
It's currently not possible
